### PR TITLE
Misc code cleanup

### PR DESCRIPTION
**Pull request by hlinnaka**
Mostly, to match upstream better. See commit messages for details.

### CODE REVIEW

File: src/backend/access/nbtree/nbtpage.c [hunk 1]
```diff
@@ -729,7 +728,7 @@ _bt_relandgetbuf(Relation rel, Buffer obuf, BlockNumber blkno, int access)
  * Lock and pin (refcount) are both dropped.
  */
 void
-_bt_relbuf(Relation rel __attribute__((unused)), Buffer buf)
+_bt_relbuf(Relation rel, Buffer buf)
```
Comment (ashwinstar) [inline src/backend/access/nbtree/nbtpage.c]: Seems should be fixed in upstream then, as is unused?

File: src/backend/optimizer/plan/createplan.c [hunk 2]
```diff
@@ -5388,7 +5387,7 @@ find_ec_member_for_tle(EquivalenceClass *ec,
  *				targetlist or insert a Result node atop the subplan to
  *				evaluate sort key exprs that are not already present in the
  *				subplan's tlist.
-x */
+ */
```
Comment (ashwinstar) [inline src/backend/optimizer/plan/createplan.c]: funny :-)

File: src/backend/access/nbtree/nbtpage.c (hunk 1 above)
Comment (hlinnaka) [reply src/backend/access/nbtree/nbtpage.c]: I suppose. It's symmetric with _bt_getbuf() to also take the Relation argument, though. I'm not a big fan of these wrappers around bufmgr.c functions in general, so perhaps they should just be ripped out and call ReadBuffer(), LockBuffer() etc. directly.

**Comment by hlinnaka** [comment]
Pushed, thanks for the review!

**Pull request merged**